src/server/dfly_bench.cc

```diff
@@ -103,6 +102,11 @@ static string GetRandomHex(size_t len) {
     }
   }
 
+  if (!ascii) {
+    for (size_t i = 0; i < len; i++) {
+      res[i] += 80;
```

Comment:
Why +80? Plz explain :smile:

Reply:
hexadecimal characters with `0` padding start at ascii `0` which is 48. +80 moves the whole string to non-ascii range.